.env.example

```diff
@@ -7,20 +7,20 @@ AWS_SECRET_ACCESS_KEY=changeme
 
 GITHUB_WEBHOOK_SECRET=test_token
 
-POSTGRES_DB=development
```

Comment:
Just checking this is meant to be deleted?

Reply:
yeh, having this set has caused issues recently, since it overwrites the default db name for both development and test, and lands them up sharing a database which is a bit weird and causes rails to behave weirdly

Reply:
the defaults of `choco_cake_development` and `choco_cake_test` are good, and make me think of tasty food which I think is a positive thing, if a little misleading

Reply:
Roger!